pkgs/by-name/os/ossia-score/package.nix

```diff
@@ -42,15 +42,15 @@
 # TODO: figure out LLVM jit
 # assert lib.versionAtLeast llvm.version "15";
 
-stdenv.mkDerivation (finalAttrs: {
```

Comment:
you can remove the `stdenv` argument at the top

Reply:
done